### PR TITLE
Configuration files for multi-year, high-topped, moist Held-Suarez

### PR DESCRIPTION
**Pull request by akshaysridhar**
Linked Issue: #1991

### CODE REVIEW

File: .buildkite/longruns/pipeline.yml [hunk 1]
```diff
@@ -272,6 +272,18 @@ steps:
         agents:
           slurm_ntasks: 32
 
+  - group: "Milestone 1.5.2a"
+    steps:
+
+      - label: ":computer: held-suarez, equilmoist, high-topped (55km), high-sponge(35km), helem_16 np_3"
+          - srun julia --project=examples examples/hybrid/driver.jl --config_file $CONFIG_PATH/longrun_hs_rhoe_equilmoist_nz64_0M_55km_rs35km
```
Comment (szy21) [inline .buildkite/longruns/pipeline.yml:279]:
```suggestion
          - srun julia --project=examples examples/hybrid/driver.jl --config_file $CONFIG_PATH/longrun_hs_rhoe_equilmoist_nz63_0M_55km_rs35km
```

File: .buildkite/longruns/pipeline.yml [hunk 2]
```diff
@@ -272,6 +272,18 @@ steps:
...
+          - julia --color=yes --project=examples post_processing/remap/remap_pipeline.jl --data_dir longrun_hs_rhoe_equilmoist_nz64_0M_55km_rs35km --out_dir longrun_hs_rhoe_equilmoist_nz64_0M_55km_rs35km
```
Comment (szy21) [inline .buildkite/longruns/pipeline.yml:280]:
```suggestion
          - julia --color=yes --project=examples post_processing/remap/remap_pipeline.jl --data_dir longrun_hs_rhoe_equilmoist_nz63_0M_55km_rs35km --out_dir longrun_hs_rhoe_equilmoist_nz63_0M_55km_rs35km
```

File: .buildkite/longruns/pipeline.yml [hunk 3]
```diff
@@ -272,6 +272,18 @@ steps:
...
+        artifact_paths: "longrun_hs_rhoe_equilmoist_nz64_0M_55km_rs35km/*"
```
Comment (szy21) [inline .buildkite/longruns/pipeline.yml:281]:
```suggestion
        artifact_paths: "longrun_hs_rhoe_equilmoist_nz63_0M_55km_rs35km/*"
```

File: config/longrun_configs/longrun_hs_rhoe_equilmoist_nz64_0M_55km_rs35km.yml [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+z_max: 55000.0
+kappa_4: 2.0e16
+vert_diff: "true"
+moist: "equil"
+precip_model: "0M"
+rayleigh_sponge: true
+alpha_rayleigh_w: 10.0
+zd_rayleigh: 35000.0
+forcing: "held_suarez"
+job_id: "longrun_hs_rhoe_equilmoist_nz64_0M_55km_rs35km"
```
Comment (szy21) [inline config/longrun_configs/longrun_hs_rhoe_equilmoist_nz64_0M_55km_rs35km.yml:17]:
```suggestion
job_id: "longrun_hs_rhoe_equilmoist_nz63_0M_55km_rs35km"
```

File: config/longrun_configs/longrun_hs_rhoe_equilmoist_nz64_0M_55km_rs35km.yml [hunk 5]
```diff
@@ -0,0 +1,17 @@
+dt_save_to_disk: "10days"
+dt: "150secs"
+t_end: "1000days"
```
Comment (szy21) [inline config/longrun_configs/longrun_hs_rhoe_equilmoist_nz64_0M_55km_rs35km.yml:3]: Let's either decrease this or increase the longrun time limit.

File: config/longrun_configs/longrun_hs_rhoe_equilmoist_nz64_0M_55km_rs35km.yml [hunk 6]
```diff
@@ -0,0 +1,17 @@
...
+z_elem: 63
+dz_bottom: 30.0
+dz_top: 3000.0
+z_max: 55000.0
+kappa_4: 2.0e16
+vert_diff: "true"
+moist: "equil"
+precip_model: "0M"
+rayleigh_sponge: true
+alpha_rayleigh_w: 10.0
```
Comment (szy21) [inline config/longrun_configs/longrun_hs_rhoe_equilmoist_nz64_0M_55km_rs35km.yml:14]: Does the simulation break with the default coefficient (1) ?

**Review by szy21 — APPROVED** [review]

File: .buildkite/longruns/pipeline.yml (hunk 3 above)
Comment (akshaysridhar) [reply .buildkite/longruns/pipeline.yml]: Pushed changes in new commit.

File: config/longrun_configs/longrun_hs_rhoe_equilmoist_nz64_0M_55km_rs35km.yml (hunk 6 above)
Comment (szy21) [reply config/longrun_configs/longrun_hs_rhoe_equilmoist_nz64_0M_55km_rs35km.yml:14]: Oh, I didn't realize we used 10 there. It's fine then.

**Comment by szy21** [comment]
Could we add a dry held suarez as well?

File: config/longrun_configs/longrun_hs_rhoe_equilmoist_nz64_0M_55km_rs35km.yml (hunk 5 above)
Comment (akshaysridhar) [reply config/longrun_configs/longrun_hs_rhoe_equilmoist_nz64_0M_55km_rs35km.yml]: Decreased t_end to 300days.

**Comment by akshaysridhar** [comment]
> 

Yes - PR against issue 1990 is on the way.

**Comment by akshaysridhar** [comment]
Day 250, Level 1 (snapshot)
[summary_1991_1_day250.pdf](https://github.com/CliMA/ClimaAtmos.jl/files/12433389/summary_1991_1_day250.pdf)

**Pull request merged**